include/k4arecord/playback.h

```diff
@@ -134,6 +134,203 @@ K4ARECORD_EXPORT k4a_result_t k4a_playback_get_calibration(k4a_playback_t playba
+ * The track name to read video settings from.
+ *
+ * \param video_settings
+ * Location to write the track's video settings.
+ *
+ * \returns ::K4A_RESULT_SUCCEEDED is returned on success, ::K4A_RESULT_FAILED is returned if the specified track does
+ * not exist or is not a video track.
+ *
+ * \xmlonly
+ * <requirements>
+ *   <requirement name="Header">playback.h (include k4arecord/playback.h)</requirement>
+ *   <requirement name="Library">k4arecord.lib</requirement>
+ *   <requirement name="DLL">k4arecord.dll</requirement>
+ * </requirements>
+ * \endxmlonly
+ */
+K4ARECORD_EXPORT k4a_result_t k4a_playback_track_get_video_settings(k4a_playback_t playback_handle,
+                                                                    const char *track_name,
+                                                                    k4a_record_video_settings_t *video_settings);
+
+/** Gets the codec id string for a particular track.
+ *
+ * The codec ID is a string that corresponds to the codec of the track's data. Some of the existing formats are listed
+ * here: https://www.matroska.org/technical/specs/codecid/index.html. It can also be custom defined by the user.
+ *
+ * \param playback_handle
+ * Handle obtained by k4a_playback_open().
+ *
+ * \param track_name
+ * The track name to read the codec id from.
+ *
+ * \param codec_id
+ * Location to write the codec id. This will be a UTF8 null terminated string. If a NULL buffer is specified, \p
+ * codec_id_size will be set to the size of buffer needed to store the string.
+ *
+ * \param codec_id_size
+ * On input, the size of the \p codec_id buffer. On output, this is set to the length of the codec_id value (including
+ * the null terminator).
+ *
+ * \returns
+ * A return of ::K4A_BUFFER_RESULT_SUCCEEDED means that the \p codec_id has been filled in. If the buffer is too small
+ * the function returns ::K4A_BUFFER_RESULT_TOO_SMALL and the needed size of the \p codec_id buffer is returned in the
+ * \p codec_id_size parameter. ::K4A_BUFFER_RESULT_FAILED is returned if the track_name does not exist. All other
+ * failures return ::K4A_BUFFER_RESULT_FAILED.
+ *
+ * \xmlonly
+ * <requirements>
+ *   <requirement name="Header">playback.h (include k4arecord/playback.h)</requirement>
+ *   <requirement name="Library">k4arecord.lib</requirement>
+ *   <requirement name="DLL">k4arecord.dll</requirement>
+ * </requirements>
+ * \endxmlonly
+ */
+K4ARECORD_EXPORT k4a_buffer_result_t k4a_playback_track_get_codec_id(k4a_playback_t playback_handle,
+                                                                     const char *track_name,
+                                                                     char *codec_id,
+                                                                     size_t *codec_id_size);
+
+/** Gets the codec context for a particular track.
+ *
+ * The codec context is codec-specific buffer that contains any required codec metadata that only known to the codec. It
```

Comment:
>The codec context is codec-specific buffer that contains any required codec metadata that only known to the codec. [](start = 3, length = 114)

nit: minor grammar fixes
... codec context is *a* codec-specific buffer ...
... metadata that is only known ...